galaxy.yml

```diff
@@ -0,0 +1,63 @@
+---
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
```

Comment:
should we just remove these comments, its pretty clear what everything is

Reply:
Removed